api/models/user.go

```diff
@@ -72,6 +73,15 @@ func GetUserByUsername(Username string) (interfaces.User, error) {
 	return userU, err
 }
 
+func GetUserByEmailAndVerifStatus(email string) (interfaces.User, error) {
```

Comment:
I don´t think this is correct. This function should only check if another user was registered with the same email, it doesn´t matter if it is or not verified.

Reply:
Oh, well, I tought that it could preserved that someone make a lot of requests to our mail. If you are not agree I will let it as the previous one.

Reply:
I'm sorry, i thought that the `GetUserByEmailAndVerifStatus` was being used in the `/signup` route.